src/Courses/Settings/OverridesForm.tsx

```diff
@@ -136,24 +201,30 @@ export const OverridesForm: React.FC<OverridesFormProps> = ({topic, userId, prob
         }
     };
 
-    const renderQuestionOverrideForm = (question: ProblemObject) => (
-        <Grid item container md={12} alignItems='flex-start' justify="center">
-            <Grid item md={4}>
-                <TextField 
-                    name="maxAttempts" 
-                    inputRef={register({
-                        required: true, 
-                        min: -1
-                    })}
-                    defaultValue={question.maxAttempts} 
-                    label='Max Attempts'
-                    type='number'
-                />
+    const renderQuestionOverrideForm = (question: ProblemObject) => {
+        // TODO enum
+        if (topic?.topicTypeId === 2) {
```

Comment:
We do currently have a `TopicTypeId` enum in the course interfaces

Reply:
Dahhh I can switch that, I wouldn't block the PR for that I can update that after the fact

Reply:
Yep, it's not a problem.